test/pkg/environment/common/expectations.go

```diff
@@ -623,75 +623,64 @@ func (env *Environment) EventuallyExpectInitializedNodeCount(comparator string,
 	Eventually(func(g Gomega) {
 		nodes = env.Monitor.CreatedNodes()
 		nodes = lo.Filter(nodes, func(n *v1.Node, _ int) bool {
-			return n.Labels[corev1beta1.NodeInitializedLabelKey] == "true"
+			return n.Labels[v1beta1.NodeInitializedLabelKey] == "true"
 		})
 		g.Expect(len(nodes)).To(BeNumerically(comparator, count))
 	}).Should(Succeed())
 	return nodes
 }
 
-func (env *Environment) EventuallyExpectCreatedNodeClaimCount(comparator string, count int) []*corev1beta1.NodeClaim {
+func (env *Environment) EventuallyExpectCreatedNodeClaimCount(comparator string, count int) []*v1beta1.NodeClaim {
 	GinkgoHelper()
 	By(fmt.Sprintf("waiting for created nodeclaims to be %s to %d", comparator, count))
-	nodeClaimList := &corev1beta1.NodeClaimList{}
+	nodeClaimList := &v1beta1.NodeClaimList{}
 	Eventually(func(g Gomega) {
 		g.Expect(env.Client.List(env.Context, nodeClaimList)).To(Succeed())
 		g.Expect(len(nodeClaimList.Items)).To(BeNumerically(comparator, count))
 	}).Should(Succeed())
-	return lo.Map(nodeClaimList.Items, func(nc corev1beta1.NodeClaim, _ int) *corev1beta1.NodeClaim {
+	return lo.Map(nodeClaimList.Items, func(nc v1beta1.NodeClaim, _ int) *v1beta1.NodeClaim {
 		return &nc
 	})
 }
 
-func (env *Environment) EventuallyExpectNodeClaimsReady(nodeClaims ...*corev1beta1.NodeClaim) {
```

Comment:
Why are we dropping this?

Reply:
Removed it since we're technically not using it, but since we're keeping everything else around, i'll add it back in.

Reply:
Got it. Yeah, could consider dropping it in a separate PR